README.md

```
@@ -1 +1,112 @@
+
+---
+
+<br/>
+
+### 🤖 사진 그룹화
+
+<!-- GIF 추가 시: <img src="assets/ai-grouping-demo.gif" width="70%" alt="AI Grouping Demo" /> -->
+![the-simpsons-homer-simpson](https://github.com/user-attachments/assets/1e85cf71-5aeb-4753-8a22-e0c57aa7e06a)
+
+Apple Vision Framework를 활용하여 비슷한 사진끼리 자동으로 분류합니다.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)